dev.alextren.Spot.json

```diff
@@ -36,6 +36,17 @@
         "*.a"
     ],
     "modules": [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
```

Comment:
This should be cleaned up, it's only needed at build time `"cleanup": ["*"]`

Reply:
Sure!